crates/gitbutler-tauri/src/menu.rs

```diff
@@ -34,6 +36,30 @@ pub async fn menu_item_set_enabled(
     Ok(())
 }
 
+#[tauri::command()]
+pub fn resolve_vscode_variant() -> &'static str {
+    let vscodium_installed = check_if_installed("codium");
+    if vscodium_installed {
+        "vscodium"
+    } else {
+        // Fallback to vscode, as it was the previous behavior
+        "vscode"
+    }
+}
+
+fn check_if_installed(executable_name: &str) -> bool {
+    let paths = match env::var_os("PATH") {
+        Some(path) => env::split_paths(&path).collect::<Vec<_>>(),
+        None => vec![],
```

Comment:
But maybe it's prettier and more 'reasonable' to bail out early here (as opposed to going with `let-else`.
```suggestion
        None => return false,
```